social_django/strategy.py

```diff
@@ -142,12 +133,8 @@ def build_absolute_uri(self, path=None):
             return path
 
     def random_string(self, length=12, chars=BaseStrategy.ALLOWED_CHARS):
-        try:
-            from django.utils.crypto import get_random_string
-        except ImportError:  # django < 1.4
-            return super(DjangoStrategy, self).random_string(length, chars)
-        else:
-            return get_random_string(length, chars)
+        from django.utils.crypto import get_random_string
```

Comment:
I know it's not actually your code, but could you move this import to the top with the rest of the django ones?

Reply:
sure thing 🙂